spec/ParseUser.spec.js

```diff
@@ -3350,7 +3350,7 @@ describe('Parse.User testing', () => {
 
   it('should not allow updates to emailVerified', done => {
     const emailAdapter = {
-      sendVerificationEmail: () => {},
+      sendVerificationEmail: () => { },
```

Comment:
Could you revert these? We usually have no space between empty curly brackets, but I think we are missing a lint rule for this

Reply:
Did some investigation and it seems there is currently no lint rule to control this; so we won't bother neither.